Cleaned up Gergo/Georgi (#46)
qmk/qmk.fm_old: _pages/keyboards/butterstick.md

---
title: "Butter Stick!"
layout: qmk
permalink: /keyboards/butterstick/
redirect_from: 
  - /butterstick/
---

# Butter Stick, 20% Ortholinear

> {% include badge.html path="Butter Stick" %}

Reviewer: I believe this should be:

```suggestion
{% include badge.html path="butterstick" %}
```

![Butter Stick](/keyboards/butterstick/flashy.jpg)

> Butter Stick is a 2x10 Split Ortho keyboard for general use based heavily on the [Georgi](/georgi/). Integrating a built in steno engine on top of QMK it's a tiny daily carry. Sporting full QWERTY support, easy chording and a well designed/extensible firmware it's a ergonomic and hacker friendly ortholinear. This is optimized for chording usage and pairs well with light switches.

Reviewer: ```suggestion
Butter Stick is a 2x10 Split Ortho keyboard for general use based heavily on the [Georgi](/georgi/). Integrating a built-in steno engine on top of QMK, it's a tiny daily carry. Sporting full QWERTY support, easy chording and a well designed/extensible firmware, it's an ergonomic and hacker-friendly ortholinear. This is optimized for chording usage and pairs well with light switches.
```

### Butter Stick is available from **[g Heavy Industries](http://gboards.ca)**.

## Steno Engine

> Butter Stick sports a onboard chording engine allowing you to define what various chords (similar to combos) do easily. This engine is the core of what makes Georgi special as it operates on chords as opposed to layers and sequences of keys. With over 263 Decillion possible combinations, a single layer is more then enough :)

Reviewer: `s/Georgi/Butter Stick/`

```suggestion
Butter Stick sports a onboard chording engine allowing you to define what various chords (similar to combos) do easily. This engine is the core of what makes Butter Stick special as it operates on chords as opposed to layers and sequences of keys. With over 263 Decillion possible combinations, a single layer is more then enough :)
```

## Gaming Mode

> Steno engine aside, there is a traditional QMK layer based option accessible from the default firmware. This allows you to access your traditional QMK layers and goodness, making for a excellent gamingpad.

Reviewer: ```suggestion
Steno engine aside, there is a traditional QMK layer-based option accessible from the default firmware. This allows you to access your traditional QMK layers and goodness, making for a excellent gaming pad.
```
